src/templates/admin/elements/review/review_meta_block.html

```diff
@@ -0,0 +1,37 @@
+<p><strong>The editor handling this piece is:</strong></p>
+
+<ul>
+    {% if review_request.article.section_editors %}
+        {% for editor in review_request.article.section_editors %}
+            <li>{{ editor }} <a href="mailto:{{ editor.email }}?subject={{ review_request.article.journal.name }} Review"><i
+                    class="fa fa-envelope"></i></a></li>
+        {% endfor %}
+    {% elif review_request.article.editors %}
```

Comment:
There could be both editors and section editors in an article no? (`if` vs `elif`)

We could also UNION (with the `.union()` method https://docs.djangoproject.com/en/1.11/ref/models/querysets/#union ) all these querysets to avoid the awkward logic.